inst/templates/methods-adcc/Luciferase/adcc-luc-lab-methods.Rmd

```diff
@@ -8,10 +8,11 @@ date: '2022-10-28'
 
 
 # Lab Methods
-CEM.NKR~CCR5~ cells [@trkola1999cell] were 
-used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]). 
+CEM.NKR~CCR5~ cells [@trkola1999cell] were used as targets for ADCC luciferase assays after infection with the infectious molecular clones ([insert IMCs]). 
```

Comment:
For the ADCC Luciferase assay, the target cells need to be transfected with the luciferase reporter gene as well. Suggest the following wording (to be confirmed by the lab):

CEM.NKR~CCR5~ cells [@trkola1999cell] transfected with the luciferase reporter gene and infected with the infectious molecular clones ([insert IMCs]) were used as targets for ADCC luciferase assays.